src/SidebarProvider.ts

```diff
@@ -0,0 +1,90 @@
+import { getNonce } from "./getNonce";
+import * as vscode from "vscode";
+import acquire
+
+
+export class SidebarProvider implements vscode.WebviewViewProvider {
+  _view?: vscode.WebviewView;
+  _doc?: vscode.TextDocument;
+
+  constructor(private readonly _extensionUri: vscode.Uri) {}
+
+  public resolveWebviewView(webviewView: vscode.WebviewView) {
+    this._view = webviewView;
+
+    webviewView.webview.options = {
+      // Allow scripts in the webview
+      enableScripts: true,
+
+      localResourceRoots: [this._extensionUri],
+    };
+
+    webviewView.webview.html = this._getHtmlForWebview(webviewView.webview);
+
+    webviewView.webview.onDidReceiveMessage(async (data) => {
+      switch (data.type) {
+        case "inputName": {
+          if (!data.value) {
```

Comment:
In the future, we can do something with this variable here - which is the candidate's name